bindings/go/CMakeLists.txt

```diff
@@ -43,7 +43,8 @@ set(go_options_file ${GO_DEST}/src/fdb/generated.go)
 
 set(go_env GOPATH=${GOPATH}
   C_INCLUDE_PATH=${CMAKE_BINARY_DIR}/bindings/c/foundationdb:${CMAKE_SOURCE_DIR}/bindings/c
-  CGO_LDFLAGS=-L${CMAKE_BINARY_DIR}/lib)
+  CGO_LDFLAGS=-L${CMAKE_BINARY_DIR}/lib
+  GO111MODULE=off)
```

Comment:
This is not actually the version of the change that we ultimately decided on; rather it is `GO111MODULE=auto`

Reply:
`off` is still in use on master and release-6.3, but `auto` is on 7.0. The `auto` change needs to be moved to master and release-6.3 as well.

Reply:
Thank for pointing out. Fixed!